src/templates/app/user/security/account_closure.jsx

```diff
@@ -0,0 +1,143 @@
+                    list_items={[
+                        { text: it.L('Go to [_1]Cashier[_2] to withdraw.', `<a href="${it.url_for('cashier')}">`, '</a>') },
+                        {
+                            id  : 'mt5_withdraw',
+                            text: it.L('Go to [_1]MT5 dashboard[_2] to withdraw from your [_3] MT5 account.', `<a href="${it.url_for('user/metatrader')}">`, '</a>', it.website_name),
+                        },
+                    ]}
+                />
+                <SeparatorLine className='gr-padding-10' />
+
+                <h2 className='primary-color'>{it.L('Reason for closure')}</h2>
+                <p>{it.L('Why do you want to close your account? (Please select one)')}</p>
+                <FormRow
+                    type='radio'
+                    id='reason'
+                    className='account-closure'
+                    label_row_class='invisible'
+                    options={[
+                        {
+                            label: it.L('Financial concerns'),
+                            value: 'financial',
+                        },
+                        {
+                            label: it.L('Too addictive'),
+                            value: 'addictive',
+                        },
+                        {
+                            label: it.L('Not interested in trading'),
+                            value: 'not interested',
+                        },
+                        {
+                            label: it.L('Prefer another trading website'),
+                            value: 'another website',
+                        },
+                        {
+                            label       : it.L('Others (please specify)'),
+                            value       : 'other',
+                            textfield_id: 'other_reason',
+                        },
+
+                    ]}
+                />
+
+                <SeparatorLine className='gr-padding-10' />
+            </div>
+            
+            <div className='invisible' id='closure_loading'>
+                <Loading />
+            </div>
+
+            <form id='form_closure'>
+                <SubmitButton
+                    text={it.L('Close my account')}
+                    custom_msg_text={it.L('Click the button below to initiate the account closure process.')}
+                    is_centered
+                    type='submit'
+                />
+            </form>
+        </div>
+    </React.Fragment>
+);
+
+const ClosureDescription = ({
+    id,
+    list_items,
+    subtitle,
+    title,
+}) => (
+    <div id={id} className='gr-padding-10'>
+        <h3 className='secondary-color'>{title}</h3>
+        <p>{subtitle}</p>
+        { list_items &&
+            <ul className='bullet'>
+                { list_items.map((item, idx) => (
+                    <li id={item.id} key={idx}>{item.text}</li>
+                ))
```

Comment:
nit, indentation

Reply:
The `eslint` is forcing the indentation, it's a little moody today 😂

Reply:
😂